Makefile

```diff
@@ -0,0 +1,3 @@
+dependency-pre-commit:
+	pre-commit install
+	pre-commit install --hook-type commit-msg
```

Comment:
delete this file, and use a symlink to the Makefile in common-dev-assets

Reply:
fixed